### PR TITLE
Shorted test project names

### PR DESCRIPTION
**Pull request by cdietrich**
Shorted test project names
Fixes: #1186

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.tests/testdata/wizard-expectations/gradle/gradle.parent/gradle/build.gradle [hunk 1]
```diff
@@ -21,14 +22,17 @@ dependencies {
...
+	inputs.file "src/main/java/mydsl/GenerateMyDsl.mwe2"
+	inputs.file "src/main/java/mydsl/MyDsl.xtext"
 	outputs.dir "src/main/xtext-gen"
-	args += "src/main/java/org/xtext/example/mydsl/GenerateMyDsl.mwe2"
+	args += "src/main/java/mydsl/GenerateMyDsl.mwe2"
 	args += "-p"
 	args += "rootPath=/${projectDir}/.."
 }
 
+test {
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/gradle/gradle.parent/gradle/build.gradle:33]: Change to Junit5 intentional?

Comment (cdietrich) [reply org.eclipse.xtext.tests/testdata/wizard-expectations/gradle/gradle.parent/gradle/build.gradle:33]: yes. wanted to have a test for that one too. see issue

File: org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoFatjar/lsMavenTychoFatjar.parent/lsMavenTychoFatjar/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+Bundle-ManifestVersion: 2
+Bundle-Name: lsMavenTychoFatjar
+Bundle-Vendor: My Company
+Bundle-Version: 1.0.0.qualifier
+Bundle-SymbolicName: lsMavenTychoFatjar; singleton:=true
+Bundle-ActivationPolicy: lazy
+Require-Bundle: org.eclipse.xtext,
+ org.eclipse.xtext.xbase,
+ org.eclipse.equinox.common;bundle-version="3.5.0"
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
```
Comment (szarnekow) [inline org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoFatjar/lsMavenTychoFatjar.parent/lsMavenTychoFatjar/META-INF/MANIFEST.MF:11]: 1.8 intentional?

Comment (cdietrich) [reply org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoFatjar/lsMavenTychoFatjar.parent/lsMavenTychoFatjar/META-INF/MANIFEST.MF:11]: yes this is a before. we have some explicit with > 8

Comment (cdietrich) [reply org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoFatjar/lsMavenTychoFatjar.parent/lsMavenTychoFatjar/META-INF/MANIFEST.MF:11]: (all to be cleaned up with java 11 move)

Comment (szarnekow) [reply org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoFatjar/lsMavenTychoFatjar.parent/lsMavenTychoFatjar/META-INF/MANIFEST.MF:11]: All good, saw the J11 case

**Review by szarnekow — APPROVED** [review]
LGTM. It's certainly possible that I missed something, but if Jenkins is happy, I'm happy too!

**Pull request merged**